diff --git a/private/post/sender.go b/private/post/sender.go
index c435264..bf5513d 100644
--- a/private/post/sender.go
+++ b/private/post/sender.go
@@ -10,6 +10,7 @@ import (
 	"net"
 	"net/mail"
 	"net/smtp"
+	"strings"
 
 	"github.com/spacemonkeygo/monkit/v3"
 	"github.com/zeebo/errs"
@@ -37,6 +38,15 @@ func (sender *SMTPSender) FromAddress() Address {
 func (sender *SMTPSender) SendEmail(ctx context.Context, msg *Message) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	for _, recipient := range msg.To {
+		address := strings.ToUpper(recipient.Address)
+		if strings.HasSuffix(address, "@EXAMPLE.COM") ||
+			strings.HasSuffix(address, "@EXAMPLE.NET") ||
+			strings.HasSuffix(address, "@EXAMPLE.ORG") {
+			return nil
+		}
+	}
+
 	client, err := smtp.Dial(sender.ServerAddress)
 	if err != nil {
 		return err
