diff --git a/satellite/admin/ui/vite.config.js b/satellite/admin/ui/vite.config.js
index 97ad7ef..755f083 100644
--- a/satellite/admin/ui/vite.config.js
+++ b/satellite/admin/ui/vite.config.js
@@ -8,7 +8,6 @@ import { existsSync, writeFileSync } from 'fs';
 import vue from '@vitejs/plugin-vue';
 import vuetify, { transformAssetUrls } from 'vite-plugin-vuetify';
 import { defineConfig } from 'vite';
-import { checker } from 'vite-plugin-checker';
 import viteCompression from 'vite-plugin-compression2';
 
 const productionBrotliExtensions = ['js', 'css', 'ttf', 'woff', 'woff2'];
@@ -32,7 +31,6 @@ const plugins = [
             configFile: 'src/styles/settings.scss',
         },
     }),
-    checker({ typescript: true, vueTsc: true }),
 ];
 
 // https://vitejs.dev/config/
