diff --git a/satellite/metabase/raw.go b/satellite/metabase/raw.go
index 96b5ab3..2fd12b9 100644
--- a/satellite/metabase/raw.go
+++ b/satellite/metabase/raw.go
@@ -161,6 +161,11 @@ func (t *TiDBAdapter) TestingDeleteAll(ctx context.Context) (err error) {
 	// isn't reset (unlike Postgres's setval); the alias cache is recreated by
 	// the caller, and tests don't depend on specific alias values.
 	_, err = t.db.ExecContext(ctx, `DELETE FROM objects; DELETE FROM segments; DELETE FROM node_aliases; DELETE FROM bucket_eventing_outbox;`)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	_, err = t.db.ExecContext(ctx, `DROP STATS segments`)
 	return Error.Wrap(err)
 }
 
